src/style.css

```diff
@@ -0,0 +1,148 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  text-decoration: none;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+/* stylings for navigation */
+nav {
+  position: absolute;
+  height: 10%;
+  width: 100%;
+  background-color: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo-container {
+  display: flex;
+  align-items: center;
+  padding-top: 2.6%;
+  padding-left: 3%;
+  color: #fff;
+  background-color: #3c3a39;
+}
+
+#logo,
+#ham {
+  background-color: #3c3a39;
+  color: #fff;
+}
+
+#logo {
+  font-family: 'Roboto', sans-serif;
+  color: white;
+}
+
+.hamberger-btn {
+  background-color: #3c3a39;
+  display: block;
+  font-size: 25px;
+  color: white;
+  float: right;
+  line-height: 80px;
+  margin-right: 40px;
+  cursor: pointer;
+}
+
+.section {
+  background-color: #1c1a19;
+  height: 100%;
+}
+
+.container {
+  position: relative;
+  background: url(Img/illustration-header-mobile.png);
+  background-repeat: no-repeat;
+
+  /* background-position: right bottom; */
+  background-position-y: 185px;
+  background-attachment: fixed;
+  background-size: 25%;
+  background-position: right bottom;
+  padding-top: 10%;
+  flex-grow: 1;
+  height: 100vh;
+}
+
+#my_name {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 535%;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  flex: none;
+  padding-top: 18%;
+  padding-left: 5%;
+  align-items: center;
+}
```

Comment:
- [ ] You've done an excellent job incorporating the requested modifications :+1:. However, it is crucial to update the font family of your main title to match the Figma template. By doing so, the title will align with the rest of the design elements and create a consistent visual identity throughout the website. This attention to detail will help to elevate the overall user experience and create a more professional-looking website.
- [ ] As previously suggested, it would be beneficial to reduce the size of the title. This will help to create a more balanced layout and ensure that the title doesn't overpower the other design elements on the page :+1:.

**Your design**

![Screenshot from 2023-04-28 15-24-30](https://user-images.githubusercontent.com/105117832/235248972-3a19e9f1-6ca0-43ca-98ed-e0b987a51a50.png)

**Figma**

![Screenshot from 2023-04-28 15-25-22](https://user-images.githubusercontent.com/105117832/235248988-61ba65ea-b3e2-45be-b89d-7dbf350dd37f.png)